articles/documentation-style-guide.md

```diff
@@ -0,0 +1,331 @@
+
+```yml
+- href: ../index.md                         
+- name: LinearDynamicalSystems              
+  items:
+  - href: lds-overview.md                    
+  - href: lds-installation-guide-windows.md
+  - href: lds-installation-guide-linux.md
+```
+
+### Manual section
+
+For the `Manual` section we typically see these elements:
+
+- Getting Started/Landing page - This typically includes a description of what the package does, how to install the package and funding/acknowledgements. This belongs in `docs/index.md`.
+- Installation Instructions - If a package requires external dependencies or additional configuration it would be helpful to dedicate an extra page to this.
+- Workflow/Conceptual Overview - Best illustrated with a flowchart or a basic workflow container. 
+
+Beyond that, there are many possible ways to organise the rest of the manual articles depending on the type of package that is being supported. 
+
+#### Individual operator articles
+
+One approach that we recommend is to try and write articles for each individual operator. This approach has several advantages:
+
+- individual operator articles can be integrated into the automatically generated `Reference` documentation using docfx's `overwrite` function, allowing for supplemental content like images and workflows.
+- These articles can also be combined into larger conceptual `Manual` articles, providing organizational flexibility.
+- Writing individual operator articles ensures complete coverage of all operators.
+
+Creating an individual operator article requires some additional steps.
+For example, to create an individual operator article for a `PredictPoses` operator that will be included in a "Network Inference" `Manual` article as well as in the automatically generated `Reference` doc:
+
+1) Create the `Bonsai_Sleap_PredictPoses.md` article and place it in the `docs/apidoc` folder. To utilize the `overwrite` function, in the markdown file, assign a UID that follows the namespace.operator format. 
+
+```yml
+---
+uid: Bonsai.Sleap.PredictPoses
+---
+Write content here.
+```
+
+2) Create a `Network-Inference.md` article and place it in the `docs/articles` folder. In the markdown file, include a reference to the individual operator.md file.
+
+```markdown
+Other 
+[!include[Title](../apidoc/Bonsai_Sleap_PredictPoses.md)]
+```
+> [!NOTE]  
+> The title is optional.
+
+### Reference section
+
+As mentioned the `Reference` section of the website is auto-generated from [XML documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags) in the code. A unique feature we have integrated into the Bonsai Editor is that when users when users right click on individual operators they will be shown the `Reference` page for that operator. For many users, this might be the most obvious entry point to the package documentation. Thus it is essential that developers ensure that:
```

Comment:
```suggestion
As mentioned the `Reference` section of the website is auto-generated from [XML documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags) in the code. We have integrated into the Bonsai Editor the ability to right-click on individual operators and select `View Help`. This will bring up and load the `Reference` page for that operator in the editor itself. For most people, this might be the most obvious entry point to the package documentation. Thus it is essential that developers ensure that:
```